packages/libs/error-stack/src/report.rs

```diff
@@ -367,6 +452,15 @@ impl<C> Report<C> {
         }
     }
 
+    /// Return the direct source frames of this report,
+    /// to get an iterator over the topological sorting of all frames refer to [`frames()`]
+    ///
+    /// [`frames()`]: Self::frames
+    #[must_use]
+    pub fn sources(&self) -> &[Frame] {
+        &self.frames
+    }
+
```

Comment:
This does not return the sources of the report but the current frames. This is considered the *current* error.